ls1intum/Artemis
# [Bugfix] Fix tutor training status in exam assessment dashboard

### Checklist

### Motivation and Context



The progress bar does not get updated in Exam Assessment Dashboard even if the tutor completed the training for the exercise. However, when the tutor navigates to Exercise Dashboard, the progress displays correct status. After investigation, I realized that wrong query is used for collecting `tutorParticipation ` in exam mode. Existing query only works for Course Assessment Dashboard. Because, the `course_id ` is empty for exercise which is belongs to exam. My solution is collecting the `tutorParticipation list`  with `examId` for exam exercises.

### Description


### Steps for Testing


1. Log in to Artemis ,Navigate to Course Management
2. Create an Exam >> Add Exercise Groups >> Add At least one exercise
3. Go to Assessment Dashboard
4. Then Open Exercise Dashboard and Start your participation for the Exercise
5. See Progress bar turns into green
6. Go back to the Assessment Dashboard and confirm that the training status is green

### Test Coverage




### Screenshots



Before Training

<img src="https://user-images.githubusercontent.com/23650630/125698464-d0a7d50d-bf60-47c0-b649-11f13a9fb778.png" width="500"/>
<img src="https://user-images.githubusercontent.com/23650630/125698497-40989a41-02f3-47ec-a732-5360a8739823.png" width="500"/>

After completing Training/Reading the Instructions

<img src="https://user-images.githubusercontent.com/23650630/125698587-e1619f55-1ff1-4d06-beb1-c64a9a0faa90.png" width="500"/>
